docs/relnotes.md

```diff
@@ -87,6 +97,29 @@ Not published yet.
 <h6 style="font-size: 1.2em"><strong><code>taipy-core</code></strong></h6>
 3.0.0
 
+- :warning: The *action* parameter of the `on_action` callback was removed for every control.<br/>
+  The signature of all *on_action8 callbacks is now unified to the following:
```

Comment:
"star on_action 8" instead of "star on_action() start"

Reply:
Thanks. Missing Shift, but no parenthesis.